PolloPollo.Entities/Application.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using PolloPollo.Shared;
+
+namespace PolloPollo.Entities
+{
+    public class Application
+    {
+        [Key]
+        public int Id { get; set; }
+
+        [Required]
+        public int UserId { get; set; }
+
+        public User User { get; set; }
+
+        [Required]
+        public int ProductId { get; set; }
+
+        public Product Product { get; set; }
+
+        [Required]
+        [StringLength(255)]
+        public string Motivation { get; set; }
+
+        [Required]
+        public DateTime TimeStamp { get; set;  }
+
+        [Required]
+        public ApplicationStatus Status { get; set; }
```

Comment:
There needs to be some logic in the DBContext similarly to the UserRole to use enums for entities.